tests/test_utils.py

```diff
@@ -0,0 +1,73 @@
+"""
+rna_cd
+Copyright (C) 2018-2019  Leiden University Medical Center, Sander Bollen
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from pathlib import Path
+from rna_cd.utils import (load_list_file, dir_to_bam_list,
+                          save_sklearn_object_to_disk,
+                          load_sklearn_object_from_disk)
+from rna_cd import VERSION
```

Comment:
Since you are testing the installed package here I would use
`pkg_resources.get_distribution("rna_cd").version`. That will get the installed package version. Not the one that lives in the `rna_cd` folder in your project.